river/compose/union.py

```diff
@@ -118,6 +120,35 @@ class TransformerUnion(base.Transformer):
     ...     ('# by place', count)
     ... )
 
+    Mini-batch example:
+
+    >>> X = [
+    ... {"place": 2, "revenue": 42},
+    ... {"place": 3, "revenue": 16},
+    ... {"place": 3, "revenue": 24},
+    ... {"place": 2, "revenue": 58},
+    ... {"place": 3, "revenue": 20},
+    ... {"place": 2, "revenue": 50},
+    ... ]
+
+    Since we need a transformer with mini-batch support to demonstrate, we shall use the `StandardScaler`
+
+    >>> from river import compose
+    >>> from river import preprocessing
+
+    >>> t_1 = compose.Select("place") | preprocessing.StandardScaler()
+    >>> t_2 = compose.Select("revenue") | preprocessing.StandardScaler()
+
+    >>> agg = compose.TransformerUnion(t_1, t_2)
+    >>> agg.learn_many(pd.DataFrame(X))
```

Comment:
This thing outputs a cumbersome dict representation, I wish there was a way to suppress the output or I could wrap it in a multi-line statement

Reply:
Yep, you can do `_ = agg.learn_many(pd.DataFrame(X))`